openrarity/cli/opensea.py

```diff
@@ -46,11 +46,37 @@ def fetch_assets(
         help="Columns to print or write to file.",
     ),
 ):
+    """
+    If we provide token_ids data as a input, using OpenseaApi, it will collect assets data. And based on the 'rank' boolean flag it will calculate ranks. At the end, based on the 'write' flag it will write the data to either stdout or to a file.
+    If we didnt provide token_ids data as a input, it uses 'start_token_id' and 'end_token_id' as a range.Using OpenseaApi, it will collect assets data. And based on the 'rank' boolean flag it will calculate ranks. At the end, based on the 'write' flag it will write the data to either stdout or to a file.
+    Parameters
+    ----------
+    token_ids_file: Path
+        Token ids file.
+    slug: str
+        Collection slug.
+    start_token_id: int,optional
+        start token_id of the collection.
+    end_token_id: int,optional
+        end token_id of the collection.
```

Comment:
Capitalization in your descriptions please. And provide an example boredapeyachtclub would be 0 and 9999 respectively.